indigo/distribution.yaml

```diff
@@ -14524,28 +14524,6 @@ repositories:
       url: https://github.com/zlt1991/tensor_field_nav.git
       version: master
     status: maintained
-  terarangerduo-ros:
-    release:
-      packages:
-      - terarangerduo
-      tags:
-        release: release/indigo/{package}/{version}
-      url: https://github.com/Terabee/terarangerduo-ros-release.git
-      version: 0.1.1-0
-    status: maintained
-  terarangerone-ros:
-    release:
-      packages:
-      - terarangerone
-      tags:
-        release: release/indigo/{package}/{version}
-      url: https://github.com/Terabee/terarangerone-ros-release.git
-      version: 0.1.1-0
-    source:
-      type: git
-      url: https://github.com/Terabee/terarangerone-ros.git
```

Comment:
@tfoote Asked me to file a PR to remove it.

Should I have just removed the release section?